README.rst

```diff
@@ -0,0 +1,188 @@
+    sudo firewall-cmd --permanent --zone dockerc --add-port 8443/tcp
+    sudo firewall-cmd --permanent --zone dockerc --add-port 53/udp
+    sudo firewall-cmd --permanent --zone dockerc --add-port 8053/udp
+    sudo firewall-cmd --reload
+
+Please also fetch the latest release of ``kontemplate`` from `here <https://github.com/tazjin/kontemplate/releases>`_ and place it somewhere where it's in your ``$PATH``.
+
+
+Developer Environment
+---------------------
+
+Please check the `cloudigrade repo<https://github.com/cloudigrade/cloudigrade#developer-environment>`_ for an up to date list of dev requirements.
+
+
+Configure AWS account credentials
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you haven't already, create an `Amazon Web Services <https://aws.amazon.com/>`_ account for **cloudigrade** to use for its AWS API calls. You will need the AWS access key ID, AWS secret access key, and region name where the account operates.
+
+Use the AWS CLI to save that configuration to your local system:
+
+.. code-block:: bash
+
+    aws configure
+
+You can verify that settings were stored correctly by checking the files it created in your ``~/.aws/`` directory.
+
+AWS access for running **cloudigrade** inside Docker must be enabled via environment variables. Set the following variables in your local environment *before* you start running in Docker containers. Values for these variables can be found in the files in your ``~/.aws/`` directory.
+
+-  ``AWS_ACCESS_KEY_ID``
+-  ``AWS_SECRET_ACCESS_KEY``
+-  ``AWS_DEFAULT_REGION``
+-  ``AWS_SQS_ACCESS_KEY_ID``
+-  ``AWS_SQS_SECRET_ACCESS_KEY``
+-  ``AWS_SQS_REGION``
+-  ``AWS_SQS_QUEUE_NAME_PREFIX``
+
+The values for ``AWS_`` keys and region may be reused for the ``AWS_SQS_`` variables. ``AWS_SQS_QUEUE_NAME_PREFIX`` should be set to something unique to your environment like ``${USER}-``.
+
+
+Common commands
+===============
+
+
+Running Locally in OpenShift
+----------------------------
+
+To start the local cluster run the following:
+
+.. code-block:: bash
+
+    make oc-up
+
+That will start a barebones OpenShift cluster that will persist configuration between restarts.
+
+If you'd like to start the cluster, and deploy Cloudigrade along with supporting services run the following:
+
+.. code-block:: bash
+
+    # When deploying cloudigrade make sure you have AWS_ACCESS_KEY_ID and
+    # AWS_SECRET_ACCESS_KEY set in your environment or the deployment will fail
+    make oc-up-all
+
+This will create the **ImageStream** to track **PostgreSQL:9.6**, create the templates for **RabbitMQ** and **cloudigrade**, and finally use the templates to create all the objects necessary to deploy **cloudigrade** and the supporting services. There is a chance that the deployment for **cloudigrade** will fail due to the db not being ready before the mid-deployment hook pod is being run. Simply run the following command to trigger a redemployment for **cloudigrade**:
```

Comment:
Should we remove that it creates the templates for RabbitMQ this since we killed the rabbit?

Reply:
Yep, good catch! 🐰🔫